docs/data/joy/components/skeleton/GeometrySkeleton.tsx

```diff
@@ -1,44 +1,17 @@
 import * as React from 'react';
 import Box from '@mui/joy/Box';
-import FormControl from '@mui/joy/FormControl';
-import FormLabel from '@mui/joy/FormLabel';
 import Skeleton from '@mui/joy/Skeleton';
-import Stack from '@mui/joy/Stack';
-import Switch from '@mui/joy/Switch';
 
 export default function GeometrySkeleton() {
-  const [loading, setLoading] = React.useState(true);
   return (
-    <Stack spacing={2} useFlexGap>
+    <div>
       <Box sx={{ m: 'auto', display: 'flex', alignItems: 'center', gap: 2 }}>
-        <Skeleton loading={loading} variant="circular" width={48} height={48} />
-        <Box>
```

Comment:
Empty boxes like this one looks wrong, I'm expanding the thought on this in https://github.com/mui/mui-x/pull/9831.